include/z64actor.h

```diff
@@ -14,6 +14,15 @@
 #define MASS_IMMOVABLE 0xFF // Cannot be pushed by OC colliders
 #define MASS_HEAVY 0xFE // Can only be pushed by OC colliders from actors with IMMOVABLE or HEAVY mass.
 
+/* These constants are the default parameters for the ubiqutous "fidgeting" formulas.
+ * Some actors call `Actor_UpdateFidgetTables` function and cache their results in `fidgetTable*` fields.
+ * Others compute them on the fly. Both variants are applied inside `*_OverrideLimbDraw` as input angles.
+ */
```

Comment:
I would prefer a simpler comment like 
```
// These are default parameters used for "animation fidgeting", which procedurally generate actor idle animations.
// These calculations may be performed within individual actors, or by using fidget tables with `Actor_UpdateFidgetTables`.
```

Reply:
Since the comment is on the constants it should say what they are for and where someone can go to find more information, like reading the comments on `Actor_UpdateFidgetTables`

Reply:
sounds great, thanks!
64d936a5a85ba24cecf146f7c17cc0d966dd110e